src/shims/unix/foreign_items.rs

```diff
@@ -267,6 +269,73 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 this.write_scalar(result, dest)?;
             }
 
+            "sched_getaffinity" => {
```

Comment:
You added this in the "allocation" group of shims. That does not seem right.  "Threading" seems best.